src/js/test/public/util.test.ts

```diff
@@ -28,12 +28,11 @@ describe("Public utils functions", function () {
       timestampDelta: BigInt(0), // 1  |
       keyLength: 0, // 1  |
       attributes: 0, // 1  |
-      length: 0, // 1  |
-      valueLen: value.length, // 1 => 11
+      valueLen: value.length, // 1 => 10
```

Comment:
What does "1 => 10" mean?

Reply:
I try to simulate the attribute bytes sum, ...1+1+1=10, but I deleted this and added a more explicit comment